### DISCUSSION TITLE
[Feature] Option to hide items that are in albums from the timeline

### DISCUSSION THREAD

**Discussion by GOvEy1nw** [immich-app/immich#13884] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I've seen others referencing hiding for the purpose of sensitivity/nsfw etc, but in a more general sense, I'd like to have a toggleable option in the settings that simply hides images from the timeline if they're in an album.

This would be similar to how plex/emby etc allow you to choose whether items that are in collections are shown individually in the library or not.

I, and I assume many others since it's implemented in other media management apps, like to use albums to categorise things (of course lol) and I like to use the timeline/default view as a way to view all the pictures I haven't categorised. I know we can already filter to view items that aren't in an album, but this feature would allow that to be more permanent for those that like it that way, without impacting the users that don't care for it.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#5158
